### ISSUE TITLE
Add ability to use Stripe emails for confirmation and invoicing

### ISSUE THREAD

**Issue by studio1902** [edalzell/statamic-charge#53]
I think atm Stripe won't send invoices when using Charge. You have to enable this on the PHP side:

https://stripe.com/docs/billing/invoices/sending

**Comment by edalzell** [comment]
@studio1902 do you need this right now? Can you use the Statamic templates?

**Comment by studio1902** [comment]
I could use the Statamic templates. The st

**Comment by edalzell** [comment]
agreed but I'm focussed on required things right now. I'll add this later.

**Comment by studio1902** [comment]
I’ll use the templates for now. Will test later tonight or tomorrow. Is it active in the current beta?

**Comment by edalzell** [comment]
yup

**Comment by studio1902** [comment]
Can't get it to work. DM'ed you :-)
